crates/autopilot/src/database/order_events.rs

```diff
@@ -27,20 +28,27 @@ impl super::Postgres {
     }
 }
 
+// Temporarily hardcoded. Will migrate to a config file in case the batches
+// approach has a positive impact.
+const DEFAULT_BATCH_SIZE: usize = 500;
```

Comment:
Did you try such a large batch size somehow?
I'm a bit worried that we'd exceed some request size limit or sth like that.

Reply:
Hm, I wouldn't say it's large. Have previous experience with 10k batches and much more params count, since batches from around 500-1000 only start showing good results. Postgres has a limit for ~65k params size in a single statement. No limit for the request size afaik.

Reply:
I just realized that all of this was in an environment with pretty smart GCs :sweat_smile:

Reply:
> Temporarily hardcoded

Famous last words